contrib/rbac-setup.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh --
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SVCCAT_NAMESPACE=catalog
```

Comment:
This assumes the namespace will be "catalog". This should probably be parameterized. A simple way would be:

```
SVCCAT_NAMESPACE="${SVCCAT_NAMESPACE:-catalog}"
```